pmd-core/src/main/java/net/sourceforge/pmd/renderers/internal/sarif/SarifLog.java

```diff
@@ -681,13 +721,15 @@ public int hashCode() {
             result = result * PRIME + ($ruleset == null ? 43 : $ruleset.hashCode());
             final java.lang.Object $priority = this.getPriority();
             result = result * PRIME + ($priority == null ? 43 : $priority.hashCode());
+            final java.lang.Object $tags = this.getTags();
```

Comment:
Similarly here, please use regular regular naming conventions (no `$`)

Reply:
It looked like this was an auto-generated class so I was just keeping it consistent. It's the same throughout all the equal methods in this class and not sure they're even being used. What would you recommend in this instance?